src/userbase-server/admin-panel/index.html

```diff
@@ -3,7 +3,7 @@
 
 <head>
   <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
+  <meta name="cyreviewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0" />
```

Comment:
What does this change do? Was it intentional?

Reply:
not intentional, removing it.